src/components/views/elements/CustomInput.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useRef, useEffect } from 'react';
+
+// via https://itnext.io/reusing-the-ref-from-forwardref-with-react-hooks-4ce9df693dd
+function useCombinedRefs(...refs) {
+    const targetRef = useRef()
+
+    useEffect(() => {
+        refs.forEach(ref => {
+            if (!ref) return
+
+            if (typeof ref === 'function') {
+                ref(targetRef.current)
+            } else {
+                ref.current = targetRef.current
+            }
+        })
+    }, [refs])
+
+    return targetRef
+}
```

Comment:
We could do this inline in `CustomInput` since we're only combining 2 refs but it's nice to have this piece of logic labeled.